test/functional/name_allowexpired.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# Public domain
```

Comment:
I'd prefer if you put an MIT header here instead as with the other files, but since this is your change, I guess "public domain" is also fine.  (Although e.g. some jurisdictions don't even allow putting stuff into the public domain, so an explicit license ensures everything is fully clear.  MIT doesn't really add any additional restrictions anyway.)

Reply:
Did you do a change to this file?  Or do you prefer to keep it as public domain rather than MIT?